ui/imports/shared/views/ProfileDialogView.qml

```diff
@@ -82,15 +82,21 @@ Pane {
         }
 
         readonly property var conns: Connections {
-            target: Global
-            function onNickNameChanged(publicKey, nickname) {
-                if (publicKey === root.publicKey) d.reload()
-            }
-            function onContactBlocked(publicKey) {
-                if (publicKey === root.publicKey) d.reload()
+            target: root.contactsStore.myContactsModel
+
+            function onItemChanged(pubKey) {
+                if (pubKey === root.publicKey)
+                    d.reload()
             }
-            function onContactUnblocked(publicKey) {
-                if (publicKey === root.publicKey) d.reload()
+        }
+
+        // FIXME: use myContactsModel for identity verification
```

Comment:
Working on it right now

Reply:
@caybro @jrainville I want to merge this PR as it and fix this connections here: https://github.com/status-im/status-desktop/pull/8791 
From user perspective is 100% working, so i prefer to make refactor in separated PR.